index.html

```diff
@@ -209,5 +193,39 @@ <h3><img src="images/Rectangle-57.png" alt=""><span>Skills</span></h3>
             </div>
         </div>
     </section> 
+    <footer class="footer">
+        <div class="footer-content">
+            <div class="footer-text">
+                <p>
+                  I'm always<br>interested in hearing about new projects, 
+                  so if you'd like to chat please get in touch.
+                </p>
+            </div>
+            <form action="https://formspree.io/f/form" id="best" method="post">
```

Comment:
Kindly remember to [change your `form_id` in the `action` property](https://formspree.io/blog/html-contact-form/), else the form will not submit the data and instead show this message:  :worried: 
![image](https://user-images.githubusercontent.com/78335447/172970911-6a978729-c656-4ade-a923-cf5a8d08a385.png)